contracts/oracle/integration/test-oracle.ava.js

```diff
@@ -40,19 +40,19 @@ test.afterEach(async (t) => {
   });
 });
 
-test("Should fail, oracle is not an issuer", async (t) => {
+test("Should fail: mint sbt, oracle is not an issuer", async (t) => {
   const {oracle_contract, claimer } = t.context.accounts;
   const mint_result = await claimer.call(oracle_contract, "sbt_mint",
     { 'claim_b64': claim_b64,
       'claim_sig' : sig_b64 },
     { attachedDeposit: NEAR.parse("0.008 N").toString() },
     { gas: Gas.parse('20 Tgas') }).catch((error) => { console.log('Transaction error:', error);});
   t.deepEqual(mint_result, {Err: 'registry.sbt_mint failed'});
-  const is_used_identity = await claimer.call(oracle_contract, "is_used_identity", { 'external_id': external_id}, { gas: 200000000000000 });
+  const is_used_identity = await claimer.call(oracle_contract, "is_used_identity", { 'external_id': external_id});
```

Comment:
this should be a `view` call, not a tx call.

Reply:
DONE